code/__DEFINES/language.dm

```diff
@@ -45,3 +45,19 @@
 #define LANGUAGE_REAGENT "reagent"
 #define LANGUAGE_MULTILINGUAL "multilingual"
 #define LANGUAGE_EMP "emp"
+
+// Languages available from Multilingual
+#define AVAILABLE_MULTILINGIAL_LANGUAGES_LIST list(\
```

Comment:
this should probably just be a global list rather than a define

`GLOBAL_INIT_VAR(name, typecacheof(list(...)))`

Reply:
done !